README.md

```diff
@@ -4,13 +4,31 @@
 | --------------------------------------------- | ----------------------------------------------- |
 | [![Main Badge][circle-master]][circle-master-base] | [![Develop Badge][circle-develop]][circle-develop-base] |
 
+- [Smart-Phase](#smart-phase)
+  - [Running Smart-Phase](#running-smart-phase)
 - [Python utility scripts](#python-utility-scripts)
 - [Docker image](#docker-image)
 - [LICENCE](#licence)
 
-Project aiming to provide a Docker image of [Smart-Phase]
+## Smart-Phase
+
+The docker image also contains [Smart-Phase]
 Please note, we do not support the Smart-Phase software, please contact the Smart-Phase repository owner.
 
+### Running Smart-Phase
+
+In order to run smart-phase in the docker image either use the command:
+
+```bash
+docker run smart-phase:latest java -jar /opt/wsi-t78/smartPhase.jar
```

Comment:
Are you going to create a `latest` tag?  We generally don't to prevent things changing without it being intentional.  It's not an issue however, you haven't added creation and push of the latest tag to the `./circleci/config.yaml`.

Reply:
Good catch, just copied my testing command.